src/test/java/com/snowflake/kafka/connector/streaming/iceberg/TestJsons.java

```diff
@@ -0,0 +1,509 @@
+package com.snowflake.kafka.connector.streaming.iceberg;
+
+/**
+ * Class to provide Iceberg schema evolution tests with schemas and payload. It decreases the size
+ * of the test file .
+ */
+class TestJsons {
+
+  public static String nestedObjectWithSchema() {
+    return "{"
+        + " \"schema\": {"
+        + "  \"type\": \"struct\","
+        + "  \"fields\": ["
+        + "   {"
+        + "    \"field\": \"object_With_Nested_Objects\","
+        + "    \"type\": \"struct\","
+        + "    \"fields\": ["
+        + "     {"
+        + "      \"field\": \"nestedStruct\","
+        + "      \"type\": \"struct\","
+        + "      \"fields\": ["
+        + "       {"
+        + "        \"field\": \"description\","
+        + "        \"type\": \"string\""
+        + "       }"
+        + "      ]"
+        + "     }"
+        + "    ]"
+        + "   }"
+        + "  ],"
+        + "  \"optional\": true,"
+        + "  \"name\": \"sf.kc.test\""
+        + " },"
+        + " \"payload\": "
+        + nestedObjectsPayload
+        + "}";
+  }
+
+  static String nestedObjectsPayload =
```

Comment:
it can be private

Reply:
Let's keep it consise with others. I don't want to restrict it's usage.